src/makie.jl

```diff
@@ -363,22 +363,30 @@ function scalarplot!(ctx, TP::Type{MakieType}, ::Type{Val{1}}, grids, parentgrid
     end
 
     function update_lines(ctx, newrange)
+	# No uniform method in Colors to create a color from either tuple or symbol
+        rgbcol = Colors.RGB{Float64}(0,0,0)
+        if ctx[:color] isa Tuple{Float64,Float64,Float64}
+          rgbcol = RGB(ctx[:color][1],ctx[:color][2],ctx[:color][3])
+        else
+          rgbcol = parse(Colors.RGB{Float64},ctx[:color])
+        end
```

Comment:
@WIAS-PDELib/pdelib-developers I am very unhappy with this part myself but Colors is not happy with assuming a tuple is RGB.
I see two ways around this depending on downstream usage.
1. Don't assume users want RGB and make them type it explicitly when supplying individual values, i.e. `color = RGB(r,g,b)`
2. Keep this somewhat ugly snippet in

Reply:
I changed the default in `dispatch.jl` to `RGB(0.0, 0.0, 0.0)` which makes the tests pass without that snippet.

Reply:
I don't think this issue is worth a breaking change. 

One possibility to avoid ugly code here might be some "MyRGBColor" in GridVisualizeTools which takes RGB, Tuples, strings, symbols.